app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    i = 0
+
+    person_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in person_list:
+        wife = people[i].get("wife")
+        husband = people[i].get("husband")
+        i += 1
```

Comment:
This is a not good approach because your person_list can be sorted or something else, so the ordering will be changed and your program will not work correct